reana_server/config.py

```diff
@@ -38,3 +38,11 @@
     'serial'
 ]
 """Available workflow engines."""
+
+ORGANIZATIONS = ['atlas', 'alice', 'cms', 'default', 'lhcb']
+
+
+ADMIN_USER_ID = "00000000-0000-0000-0000-000000000000"
```

Comment:
remember to change this later...

Reply:
probably we should get it from env and inject it as a secret into the container (means PR on `reana-cluster` and docs update)? `REANA_SERVER_ADMIN_TOKEN`? cc @dinosk